app/javascript/components/gtl/DataTable.jsx

```diff
@@ -50,7 +50,12 @@ export const DataTable = ({
              name={'selectAll'}
              key={'selectAll'}
              checked={!!checkedItems['selectAll']}
-             onChange={event => localOnSelectAll(rows, event)}
+             onChange={localOnSelectAll}
+             onKeyPress={localOnSelectAll}
+             role="checkbox"
+             aria-checked="false"
+             tabIndex="0"
+             aria-labelledby={'selectAll'}
```

Comment:
This is still not valid and the syntax is redundant. Where is this `label#selectAll` element you're trying to refer to?

Reply:
@skateman there is no label on the checkboxes, i used the name of the element, not sure what i should use there in this case.

Reply:
Then please skip this attribute

Reply:
@h-kataria workaround for this issue is, you can add a label with id `selectAll` and can hide it using css.

Reply:
You'd need to generate the IDs for these hidden fields to avoid duplication...

Reply:
One more tip is, along with css, you can add aria-hidden="true" on the element , otherwise screen reader will read this text randomly and it will confuse people as we don't have that on screen.

Reply:
@skateman please re-review.

Reply:
@kavyanekkalapu please take a look, made suggested changes, added label around checkboxes along with css to hide labels